docker/pg_tests/scripts/tests/daemon_test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+set -e -x
+
+CONFIG_FILE="/tmp/configs/wal_perftest_config.json"
+
+COMMON_CONFIG="/tmp/configs/common_config.json"
+TMP_CONFIG="/tmp/configs/tmp_config.json"
+cat ${CONFIG_FILE} > ${TMP_CONFIG}
+
+echo "," >> ${TMP_CONFIG}
+cat ${COMMON_CONFIG} >> ${TMP_CONFIG}
+/tmp/scripts/wrap_config_file.sh ${TMP_CONFIG}
+
+WAL=$(ls -l ${PGDATA}/pg_wal | head -n2 | tail -n1 | egrep -o "[0-9A-F]{24}")
+
+wal-g --config=${TMP_CONFIG} daemon "${PGDATA}"/pg_wal
+
+# shellcheck disable=SC2039
+nc -U /tmp/wal-push.sock <<< 'CHECK'
+sleep 1
```

Comment:
It is better to check for reply from the socket and remove the `sleep 1` calls. It would help to:
- make sure that WAL-G responded correctly
- avoid race conditions when using the `sleep 1` logic